src/Model/RouteRepository.php

```diff
@@ -16,33 +16,45 @@
 
 class RouteRepository implements RouteRepositoryInterface
 {
-
-    /**
-     * @var CollectionFactory
-     */
+    /**@var CollectionFactory */
     private $collectionFactory;
 
-    /** @var []RouteInterface */
+    /** @var array */
```

Comment:
This is actively throwing away useful type information. Please revert changes like these.